### PR TITLE
rewrite command line presentation

### PR DESCRIPTION
**Pull request by showalrm**
Finally got around to making this PR again. I'm open to suggestions on this.

Resolves #40

### CODE REVIEW

File: CommandLine.md [hunk 1]
```diff
@@ -1,92 +1,253 @@
...
+
+- The up and down arrows can be used to cycle through recent commands.
+- The tab key will complete a command
+- `history` will show the most recent commands
+- Ctrl-r will search through the history
+- The current directory can be referred to as `.` and the parent directory can 
+be referred to as `..`
+- The home directory can be referred to as `~`
+- The last command can be referred to with `!!`, which is especially useful if 
+  you forget to add `sudo` in front of a command that needs priveledge elevation.
```
Comment (Advill) [inline CommandLine.md]:
```suggestion
  you forget to add `sudo` in front of a command that needs privilege elevation.
```

File: CommandLine.md [hunk 2]
```diff
@@ -1,92 +1,253 @@
...
+redirect input and output. Also note that cat is short for "concatenate", 
+because you can specify multiple files and it will concatenate them.
 
+#### Standard Out
+
+Standard out, shortened to stdout, is sometimes the terminal window, but it 
+can also be a file. If you type `echo "test"`, the output will be sent to the
+terminal window. `echo "test" > test.txt` will output to the specified file. If 
+you'd like to append a file rather than overwriting it, try `echo "test" >> 
+test.txt"
```
Comment (Advill) [inline CommandLine.md]:
```suggestion
test.txt"`
```

File: CommandLine.md [hunk 3]
```diff
@@ -1,92 +1,253 @@
...
+
+### Variables
+
+Just like any programming language, you can use variables. There aren't any
+variable types, and there's no need to declare a variable before you assign it.
+
+```bash
+#!/bin/bash
+
+$STR="Hello, world!"
```
Comment (ripleymj) [inline CommandLine.md]: STR="Hello, world!"

**Comment by ripleymj** [comment]
Same situation as @Advill. It's good on first glance, but I haven't had the time to review as detailed as I'd like.

**Pull request merged**